src/main/java/edu/kit/kastel/sdq/artemis4j/client/ArtemisClient.java

```diff
@@ -0,0 +1,201 @@
+package edu.kit.kastel.sdq.artemis4j.client;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.StreamReadFeature;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import edu.kit.kastel.sdq.artemis4j.ArtemisNetworkException;
+import okhttp3.Cookie;
+import okhttp3.CookieJar;
+import okhttp3.HttpUrl;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class ArtemisClient {
+    public static final ObjectMapper MAPPER = createObjectMapper();
+
+    private static final Logger log = LoggerFactory.getLogger(ArtemisClient.class);
+    private static final String COOKIE_NAME_JWT = "jwt";
+    private static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+
+    private final ArtemisInstance artemis;
+    private final String jwtToken;
+    private final String password;
```

Comment:
Why do you store the password? The jwtToken should access all

Reply:
It's needed for cloning from LocalVC. The token doesn't work there